README.md

```diff
@@ -54,12 +102,15 @@ You can override standard build using docker build arguments
 Build frontends on the main branch and contact frontend on a custom branch `hantt12-patch-1`
 
 ```sh
-docker build --build-arg CONTACTS_GIT_BRANCH=hantt12-patch-1 -t openpaas-front .
+# Using BuildKit
+DOCKER_BUILDKIT=1 docker build --pull -f Dockerfile.dev --build-arg CONTACTS_GIT_BRANCH=hantt12-patch-1 -t openpaas-front-dev . --no-cache
```

Comment:
Should change `CONTACTS_GIT_BRANCH` to `CONTACTS_GIT_TREEISH`

Reply:
good catch i change documentation